projects/NETCore.md

```diff
@@ -17,6 +17,6 @@
 * [Contribute](https://github.com/dotnet/corefx/blob/master/CONTRIBUTING.md)
 * [Discussions](http://forums.dotnetfoundation.org/)
 * [Samples](https://github.com/Microsoft/dotnetsamples)
-* [Blog](http://blogs.msdn.com/b/dotnet/)
+* [Blog](http://blogs.msdn.microsoft.com/dotnet/)
 * Twitter: [@DotNet](https://twitter.com/dotnet)
-* [[WIP] Documentation](http://dotnet.readthedocs.org/)
+* [Documentation](http://docs.microsoft.com/dotnet/core/index)
```

Comment:
also wondering if we want to link to core directly as in here or to https://docs.microsoft.com/dotnet/